keyboards/skyloong/gk75/q2/q2.c

```diff
@@ -0,0 +1,294 @@
+//  Copyright 2023 NaturalZh (@NaturalZh)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+
+int FN_ON = 0;
+bool WIN_LOCK = 0;
+bool FN_WIN = 0;
+bool FN_MAC = 0;
+bool L_WIN = 0;
+bool L_MAC = 0;
+
+#if defined(RGB_MATRIX_ENABLE)  /*&& defined(CAPS_LOCK_INDEX)*/
+const aw20216s_led_t PROGMEM g_aw20216s_leds[AW20216S_LED_COUNT] = {
+/* Each AW20216 channel is controlled by a register at some offset between 0x00
+ * and 0xD7 inclusive.
+ * See drivers/awinic/aw20216.h for the mapping between register offsets and
+ * driver pin locations.
+ *   driver
+ *   |  R location
+ *   |  |          G location
+ *   |  |          |          B location
+ *   |  |          |          | */
+    {0, CS1_SW1,   CS2_SW1,   CS3_SW1},  //ESC
```

Comment:
All of these are using the deprecated defines. Where the current format is `SW1_CS1`.

Reply:
Okey, It`s updated. Please check it again @zvecr